phoenix-ios/phoenix-ios.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -27,6 +27,15 @@
         "version" : "0.4.1"
       }
     },
+    {
+      "identity" : "codablecsv",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/dehesa/CodableCSV.git",
+      "state" : {
+        "revision" : "99ace2cfdfbc19108b529c021005fb57a460e715",
+        "version" : "0.6.7"
+      }
```

Comment:
I think we should try to limit our external dependencies, for maintenance and security reasons. The [RFC](https://www.ietf.org/rfc/rfc4180.txt) seem simple enough and the volume of data Phoenix exports is not so large that we need to use a specialised library. Also we know what format we use so our implementation can be simplistic.

Reply:
Done in [5887637](https://github.com/ACINQ/phoenix/pull/337/commits/5887637bfd95bd07bdd1671fa9d0b38a1063c6b1)